reporter/src/main/kotlin/gitlab/GitLabLicenseModelMapper.kt

```diff
@@ -53,7 +53,7 @@ private fun Collection<Package>.toLicenses(): List<License> {
         License(
             id = singleLicenseExpression.simpleLicense(),
             name = singleLicenseExpression.toLicenseName(),
-            url = "" // TODO
+            url = if (singleLicenseExpression.isValid()) singleLicenseExpression.getLicenseUrl() else ""
```

Comment:
I think the validity check should be inside the `getLicenseUrl` function. With the changes proposed below this could become:
```
url = singleLicenseExpression.getLicenseUrl().orEmpty()
```

Reply:
Done.